ext/opcache/ZendAccelerator.h

```diff
@@ -278,6 +281,8 @@ typedef struct _zend_accel_shared_globals {
 	void *jit_traces;
 	const void **jit_exit_groups;
 
+	size_t map_ptr_static_last;
```

Comment:
I hope no one is doing anything funky ABI-wise, they shouldn't of course...
Anyway, since ABI _shouldn't_ be a concern, can we please put this next to the `map_ptr_last` field? A bit more cache locality.